docs/families/btc/integration_14_explorer.md

```diff
@@ -0,0 +1,18 @@
+# Change Blockchain Explorer Params {#integration_14_explorer}
+
+[TOC]
+
+# Overview
+
+Clone https://github.com/VeriBlock/vbk-ri-btc-explorer repo with explorer source code.
+
+Create json file at `app/coins/youraltcain.js` (you should get other files as example). This file contain params of your altchain. Fill that file with genesis, chain and other params of your altchain. For example to get info from genesis coinbase TX you can print HEX encoded TX data to log in genesis block creation function (`CreateGenesisBlock`) and then use RPC `decoderawtransaction` to get info from this TX.
```

Comment:
This file contains params

Reply:
"For example to get info from genesis coinbase TX you can print HEX encoded TX data" - do we have any better way of configuring coin params?